### PR TITLE
[WIP][v6r20] HowTo section for the doc

### PR DESCRIPTION
**Pull request by chaen**
This is just the beginning of the HowTo we discussed long ago. @fstagni maybe that would be a good place to write (or at least link) the docs you wrote for *adding a site* or *adding a queue*

BEGINRELEASENOTES
*Docs
NEW: DataManagement How-To

ENDRELEASENOTES

### CODE REVIEW

File: docs/source/HowTo/DataManagement/index.rst [hunk 1]
```diff
@@ -0,0 +1,191 @@
+==============
+DataManagement
+==============
+
+
+For an introduction about DataManagement concepts, please see the :ref:`introduction <dms-concepts>`
+
+All the commands mentionned bellow can accept several StorageElements and LFNs as parameters. Please use `--help` for more details.
```
Comment (fstagni) [inline docs/source/HowTo/DataManagement/index.rst]: mentioned below

**Comment by fstagni** [comment]
Did you know about https://dirac.readthedocs.io/en/latest/UserGuide/Tutorials/DataManagementBasic/index.html ? It's basically a user tutorial and there are repetitions, so 2 should be integrated. There's also https://dirac.readthedocs.io/en/latest/UserGuide/Tutorials/DataManagementAdvanced/index.html

**Comment by chaen** [comment]
No I did not know, thanks.
I think I will remove what is in the tutorial, since it looks more like a HowTo. What do you think ?

**Comment by chaen** [comment]
@andresailer sorry I know I asked already but I can't remember. Wasn't there at some point this idea of reviving the automatic script documentation ?

**Comment by fstagni** [comment]
I am fine with such a re-organization, just we should agree in which section this goes: in the user section or in the admin one. This is probably user.

**Comment by andresailer** [comment]
> @andresailer sorry I know I asked already but I can't remember. Wasn't there at some point this idea of reviving the automatic script documentation ?

I also don't remember if you asked this before. In #3844 I made a big step towards automatically running `buildScriptsDocs`. The goal would be to have `buildScriptsDocs` not produce any changes in the docs, unless the commands themselves were changed. This needs fixes in a few docstrings, like fixing some indendation, and ensuring that there are warnings produced if a script is not part of any overview page. Now that the documentation CI will actually fail (in integration) when this happens, this will be done soon.

File: docs/source/HowTo/DataManagement/index.rst [hunk 2]
```diff
@@ -0,0 +1,191 @@
...
+            CERN-USER : srm://srm-eoslhcb.cern.ch:8443/srm/v2/server?SFN=/eos/lhcb/grid/user/lhcb/user/c/chaen/diracTutorial.txt
+             RAL-USER : srm://srm-lhcb.gridpp.rl.ac.uk:8443/srm/managerv2?SFN=/castor/ads.rl.ac.uk/prod/lhcb/user/c/chaen/diracTutorial.txt
+
+
+How to get the xroot URL for my LFN
+-----------------------------------
+
+In order to get an xroot URL usable from for example ROOT::
+
+    [DIRAC prod] chaen $ dirac-dms-lfn-accessURL --Protocol=root,xroot /lhcb/user/c/chaen/diracTutorial.txt RAL-USER
```
Comment (andresailer) [inline docs/source/HowTo/DataManagement/index.rst]: While trying to automate the command reference building, and looking at the `--help` output from dirac-dms-lfn-accessURL, then --Protocol is not an option, but an optional positional argument.

**Comment by fstagni** [comment]
Update on this? Maybe target v6r21?

**Pull request closed without merging**